free-2.md

```diff
@@ -46,6 +59,7 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+[This website](http://www.matchingfoodandwine.com/) seems to do exactly what this DSL describes, but I played around with it and it seems really clunky and limited in the food options. It also simply links to individual blogger opinions on various wines, and doesn't do any sort of "objective" calculation of which flavors in the food would go well with different wines.
```

Comment:
Are there "objective" calculations of which flavors go well with different wines? Since I know very little about wine, I cannot tell how much of these suggestions are _opinion_ and how much are _fact_.